tests/integration/test_solver_slippage.py

```diff
@@ -0,0 +1,206 @@
+            f"v1_not_v2={len(self.v1_not_v2)}\n"
+            f"v2_not_v1={len(self.v2_not_v1)}\n"
+            f"overlap={len(self.overlap)}"
+        )
+
+    def to_json(self, tx_hash: str):
+        return json.dumps({"V1": self.v1[tx_hash], "V2": self.v2[tx_hash]})
+
+
+class TestQueryMigration(unittest.TestCase):
+    def setUp(self) -> None:
+        self.dune = DuneClient(os.environ["DUNE_API_KEY"])
+        self.period = AccountingPeriod("2022-11-01", length_days=7)
+        self.slippage_query = QUERIES["PERIOD_SLIPPAGE"]
+        self.parameters = self.period.as_query_params()
+        self.writer = FileIO(FILE_OUT_DIR)
+
+    def exec_or_get(self, query: Query, result_id: Optional[str] = None):
+        if not result_id:
+            return self.dune.refresh(query)
+        return self.dune.get_result(result_id)
+
+    def get_cte_rows(
+        self,
+        cte_name: str,
+        tx_hash: Optional[str] = None,
+        v1_cache: Optional[str] = None,
+        v2_cache: Optional[str] = None,
+    ):
+        parameters = self.period.as_query_params()
+        parameters.append(QueryParameter.enum_type("CTE_NAME", cte_name))
+        if tx_hash:
+            parameters.append(QueryParameter.text_type("TxHash", tx_hash))
+
+        v1_results = self.exec_or_get(Query(1570227, params=parameters), v1_cache)
+        v2_results = self.exec_or_get(Query(1570561, params=parameters), v2_cache)
+
+        v1_rows = v1_results.get_rows()
+        v2_rows = v2_results.get_rows()
+        return v1_rows, v2_rows
+
+    def get_cte_results(
+        self,
+        cte_name: str,
+        tx_hash: Optional[str] = None,
+        v1_cache: Optional[str] = None,
+        v2_cache: Optional[str] = None,
+    ):
+        # tokens.erc20 broken because of: https://github.com/duneanalytics/spellbook/pull/2083
+        # Had to make query forks.
+        v1_rows, v2_rows = self.get_cte_rows(cte_name, tx_hash, v1_cache, v2_cache)
+        self.writer.write_csv(v1_rows, f"{cte_name}_{tx_hash}_v1.csv")
+        self.writer.write_csv(v2_rows, f"{cte_name}_{tx_hash}_v2.csv")
+        return Comparison.from_dune_results(v1_rows, v2_rows)
+
+    def test_batch_transfers(self):
+        table_name = "batch_transfers"
+        data = self.get_cte_results(
+            table_name,
+            v1_cache="01GJQNSER0PYSMGHMEBDT03805",
+            v2_cache="01GJQNSER0PYSMGHMEBDT03805",
+        )
+
+        # No missing transactions.
+        self.assertEqual(data.v2_not_v1, set())
+        self.assertEqual(data.v1_not_v2, set())
+
+        for tx_hash in data.overlap:
+            x = data.v1[tx_hash]
+            y = data.v2[tx_hash]
+            self.assertEqual(
+                len(x),
+                len(y),
+                f"{table_name} failed at {tx_hash} on\n {data.to_json(tx_hash)}",
+            )
+
+    def test_excluded_batches(self):
+        data = self.get_cte_results("excluded_batches")
+        self.assertEqual(len(data.v1), len(data.v2))
```

Comment:
This test is basically useless and trivial in comparison with the others. It is not worth having around.
```suggestion
```